presto-main/src/test/java/com/facebook/presto/sql/planner/iterative/rule/BenchmarkReorderJoins.java

```diff
@@ -0,0 +1,118 @@
+/*
```

Comment:
See my comments about cross join enumeration in sub-graphs. I think when we get rid of that then times for 10 tables would be much better.